src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1797,6 +1800,37 @@ public TransformReason get(EntityTransformEvent event) {
 				return event.getTransformReason();
 			}
 		}, EventValues.TIME_NOW);
+
+		// PlayerLeashEntityEvent
+		EventValues.registerEventValue(PlayerLeashEntityEvent.class, Player.class, new Getter<Player, PlayerLeashEntityEvent>() {
```

Comment:
might be worth documenting here that a Player event value is explicitly registered as PlayerLeashEntityEvent does not extend PlayerEvent